modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -59,7 +77,7 @@ $ oc -n openshift-nmstate delete nmstate nmstate
 $ oc delete --all deployments --namespace=openshift-nmstate
 ----
 
-. Delete all the custom resource definition (CRD), such as `nmstates`, that exist in the `nmstate.io` namespace by running the following commands:
+. Delete all the custom resource definitions (CRDs), such as `nmstates`, that exist in the `nmstate.io` namespace by running the following commands:
```

Comment:
1. There is no such namespace `nmstate.io`. nmstate is installed at `openshift-nmstate` namespace.
2. CRDs (as opposed to CRs) are cluster-scoped resources and are not belong to a specific namespace.
```suggestion
. Delete all the custom resource definitions (CRDs), such as `nmstates.nmstate.io` by running the following commands:
```

Reply:
:heavy_check_mark: